application/controllers/users.py

```diff
@@ -23,12 +34,181 @@
     help='user_name parameter is required'
 )
 user_post_parser.add_argument(
-    'token',
+    'email',
+    type=str,
+    required=True,
+    location=['json'],
+    help='email parameter is required'
+)
+
+timer_post_parser = reqparse.RequestParser()
+timer_post_parser.add_argument(
+    'work_interval',
+    type=str,
+    required=True,
+    location=['json'],
+    help='work_interval parameter is required'
+)
+timer_post_parser.add_argument(
+    'rest_interval',
     type=str,
     required=True,
     location=['json'],
-    help='token parameter is required'
+    help='rest_interval parameter is required'
+)
+timer_post_parser.add_argument(
+    'sound',
+    type=str,
+    required=True,
+    location=['json'],
+    help='sound parameter is required'
+)
+timer_post_parser.add_argument(
+    'mood',
+    type=bool,
+    required=True,
+    location=['json'],
+    help='mood parameter is required'
+)
+timer_post_parser.add_argument(
+    'user_id',
+    type=int,
+    required=True,
+    location=['json'],
+    help='user_id parameter is required'
+)
+
+rest_fields = {
+    'id': fields.Integer,
+    'mood_rating_1': fields.Integer,
+    'mood_rating_2': fields.Integer,
+    'content_selected': fields.String,
+    'focus_interval': fields.String,
+    'rest_interval': fields.String,
+    'user_id': fields.Integer
+}
+
+rest_list_fields = {
+    'count': fields.Integer,
+    'rests': fields.List(fields.Nested(rest_fields)),
+}
+
+rest_post_parser = reqparse.RequestParser()
+rest_post_parser.add_argument(
+    'mood_rating_1',
+    type=int,
+    required=True,
+    location=['json'],
+    help='init mood parameter is required'
 )
+rest_post_parser.add_argument(
+    'mood_rating_2',
+    type=int,
+    required=True,
+    location=['json'],
+    help='end mood parameter is required'
+)
+rest_post_parser.add_argument(
+    'content_selected',
+    type=str,
+    required=True,
+    location=['json'],
+    help='content parameter is required'
+)
+rest_post_parser.add_argument(
+    'focus_interval',
+    type=str,
+    required=True,
+    location=['json'],
+    help='focus interval parameter is required'
+)
+rest_post_parser.add_argument(
+    'rest_interval',
+    type=str,
+    required=True,
+    location=['json'],
+    help='rest interval parameter is required'
+)
+rest_post_parser.add_argument(
+    'user_id',
+    type=int,
+    required=True,
+    location=['json'],
+    help='user_id parameter is required'
+)
+
+class UserTimerResource(Resource):
+    def get(self, user_id=None):
+        if user_id:
+            timer = Timer.query.filter_by(user_id=user_id).first()
+            return marshal(timer, timer_fields)
+
+    @marshal_with(timer_fields)
+    def put(self, user_id=None):
+        if user_id:
```

Comment:
Not that I know Python, but this looks good! Great workaround for creating/updating a timer